### DISCUSSION TITLE
Error while trying PatchListingItems

### DISCUSSION THREAD

**Discussion by moschkowitz** [abuzuhri/Amazon-SP-API-CSharp#560] [Q&A]
I get this error: 
ListingsItemPutRequest is a required property for ParameterPatchListingItem and cannot be null
 Arborescence des appels de procédure :
   à FikaAmazonAPI.Parameter.ListingItem.ParameterPatchListingItem.Check()
   à FikaAmazonAPI.Services.ListingsItemService.<PatchListingsItemAsync>d__8.MoveNext()
...

I don't get why "ListingsItemPutRequest" is required and it is not a property of ParameterPatchListingItem

**Comment by TechyChap** [comment]
It's a typo in the library. 

https://github.com/abuzuhri/Amazon-SP-API-CSharp/blob/de1d9a0dacd84288b54f06473a8066b666fca96b/Source/FikaAmazonAPI/Parameter/ListingsItems/ParameterPatchListingItem.cs#L29

Should read "ListingsItemPatchRequest is a required property for ParameterPatchListingItem and cannot be null"

You need to create a collection of one or more patches and pass them into this parameter.

https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-use-case-guide#tutorial-partially-update-a-listing
